typing_inspect.py

```diff
@@ -186,6 +186,16 @@ def is_classvar(tp):
     return type(tp) is _ClassVar
 
 
+def is_new_type(tp):
+    """Tests if the type represents a type alias. Examples::
```

Comment:
"type alias" is not a correct term for this. The PEP 484 defines it as "Distinct type", also just saying "a NewType" is fine.

Reply:
changed to `distinct type` alternative. :+1: